tests/ert/unit_tests/scheduler/test_slurm_driver.py::test_exit_codes

### What happened? (You can include a screenshot if it helps explain)

```
FAILED tests/ert/unit_tests/scheduler/test_slurm_driver.py::test_exit_codes[echo 0-false-echo "COMPLETED|0:0"-0] - Failed: Timeout (>10.0s) from pytest-timeout.
```

Fails when attempting to `git push`

Sometimes passes when `uv run just rapid-tests`

Always passes when running `uv run pytest tests/ert/unit_tests/scheduler/test_slurm_driver.py::test_exit_codes -p no:qt`

### What did you expect to happen?

_No response_

### Steps to reproduce

```console

```

### Environment where bug has been observed

- [ ] Python 3.11
- [ ] Python 3.12
- [x] macOS
- [ ] RHEL8
- [ ] RHEL9
- [ ] Local queue
- [ ] LSF queue
- [ ] Slurm queue
- [ ] OpenPBS queue

--- a/tests/ert/unit_tests/scheduler/test_slurm_driver.py
+++ b/tests/ert/unit_tests/scheduler/test_slurm_driver.py
@@ -39,6 +39,7 @@ def capturing_sbatch(monkeypatch, tmp_path):
     sbatch_path.chmod(sbatch_path.stat().st_mode | stat.S_IEXEC)
 
 
+@pytest.mark.skip_mac_ci
 @pytest.mark.usefixtures("use_tmpdir")
 @pytest.mark.parametrize(
     ("sbatch_script", "scontrol_script", "sacct_script", "exit_code"),

